pkg/controller/core.oam.dev/v1alpha2/applicationconfiguration/render.go

```diff
@@ -347,6 +347,16 @@ func setTraitProperties(t *unstructured.Unstructured, traitName, namespace strin
 	if t.GetName() == "" {
 		t.SetName(traitName)
 	}
+
+	if controller := metav1.GetControllerOf(t); controller != nil {
```

Comment:
Does cluster-scope workload also need this logic?

Reply:
Currently, that's the easiest way to judge it's a cluster-scoped resource. I just avoid to involve `discovermapper` pkg method here to keep minimal changes.

Reply:
https://github.com/oam-dev/kubevela/blob/a86ec24ee477db0d9360ce563513065de824a205/pkg/controller/core.oam.dev/v1alpha2/applicationconfiguration/render.go#L203.  I mean current component render logic would set namespace and ownerRef for workload too. Does this matter for cluster-scope workload?

Reply:
The function name is `setTraitProperties`, I think it only works for trait.

Reply:
So, current cannot support cluster-scope workload?